ipv-core/src/test/java/com/ibm/whc/deid/providers/masking/BinningMaskingProviderTest.java

```diff
@@ -159,4 +160,53 @@ public void testStartValueNegative() {
     assertEquals("7:13", maskingProvider.mask("7"));
     assertEquals("7:13", maskingProvider.mask("7.0001"));
   }
+  
+  @Test
+  public void testConstantValues() {
+    BinningMaskingProviderConfig config = new BinningMaskingProviderConfig();  
```

Comment:
This test is good.  You might want to add another test where only the lower threshold is active and another where only the upper is active to make sure they will work independently.